src/sasquatchbackpack/sasquatch.py

```diff
@@ -0,0 +1,134 @@
+"""Handles dispatch of backpack data to kafka."""
+
+__all__ = ["BackpackDispatcher", "DispatcherConfig"]
+
+import os
+from dataclasses import dataclass
+from string import Template
+
+import requests
+
+from sasquatchbackpack import sources
+
+# Code yoinked from https://github.com/lsst-sqre/
+# sasquatch/blob/main/examples/RestProxyAPIExample.ipynb
+
+
+@dataclass
+class DispatcherConfig:
+    """Class containing relevant configuration information for the
+    BackpackDispatcher.
+
+    Values
+    ------
+    sasquatch_rest_proxy_url
+        environment variable contatining the target for data
+    partitions_count
+        number of partitions to create
+    replication_factor
+        number of replicas to create
+    namespace
+        environment varible containing the target namespace
+    """
+
+    sasquatch_rest_proxy_url = os.getenv(
+        "SASQUATCH_REST_PROXY_URL",
+        "https://data-int.lsst.cloud/sasquatch-rest-proxy",
+    )
+    partitions_count = 1
+    replication_factor = 3
+    namespace = os.getenv("BACKPACK_NAMESPACE", "lsst.backpack")
+
+
+class BackpackDispatcher:
+    """A class to send backpack data to kafka.
+
+    Parameters
+    ----------
+    source
+        DataSource containing schema and record data to be
+        posted to remote
+    config
+        DispatcherConfig to transmit other relevant information to
+        the Dispatcher
+    """
+
+    def __init__(
+        self, source: sources.DataSource, config: DispatcherConfig
+    ) -> None:
+        self.source = source
+        self.config = config
+        self.schema = Template(source.load_schema()).substitute(
+            {
+                "namespace": self.config.namespace,
+                "topic_name": self.source.topic_name,
+            }
+        )
+
+    def create_topic(self) -> str:
+        """Create kafka topic based off data from provided source.
+
+        Returns
+        -------
+        str
+            The results of the POST request in string format
```

Comment:
Note that this function is doing a GET request first. 
Therefor this message can be a little more generic.